doc/nrf/drivers/nrf700x.rst

```diff
@@ -33,8 +33,11 @@ Design overview
 The nRF700X Wi-Fi driver follows an OS agnostic design, and the driver is implemented into OS agnostic and OS (Zephyr) specific implementation files.
```

Comment:
```suggestion
The nRF700X Wi-Fi driver follows an OS agnostic design, and the driver implementation is divided into:
- OS agnostic code and,
- OS (Zephyr) specific code.

```

Reply:
Don't want to highlight the OS agnostic part, hence put it within the sentence.

Reply:
If this suggestion is implemented, please use `*` to create bullet points, and include a blank line between the two points.

Reply:
@krish2718 : Ok understood. In that case I guess we can just rephrase the sentence as `The nRF700X Wi-Fi driver follows an OS agnostic design, and the driver implementation is split into OS agnostic and OS (Zephyr) specific code.` ?